src/nni_manager/common/trainingService.ts

```diff
@@ -8,6 +8,8 @@
  */
 type TrialJobStatus = 'UNKNOWN' | 'WAITING' | 'RUNNING' | 'SUCCEEDED' | 'FAILED' | 'USER_CANCELED' | 'SYS_CANCELED' | 'EARLY_STOPPED';
 
+type LogType = 'TRIAL_LOG' | 'TRIAL_STDERR';
```

Comment:
Does TRIAL_LOG means stdout log?

Reply:
Yes, corresponding to `trial.log`.

Reply:
I think we discussed and decided that it should be ERROR_MESSAGE

Reply:
change to `TRIAL_LOG`  and `TRIAL_ERROR`